README.md

```diff
@@ -2,6 +2,16 @@
 
 In this challenge, you'll be **styling and validating a sign up form**, a common use case for client-side validation.
 
+### Learning Outcomes
+Understand:
+
+* what client side validation is
```

Comment:
I think this could be a little more detailed.
Suggestion:

- the UX benefits of client side validation
- html5 validation
- different type of focus events

Reply:
It would be good to see this workshop also address the UX difficulties in patterns like this. e.g. validate-on-blur is distracting and a bit shit, especially if you click into an input then click out without typing anything.

Also a11y is an important part of this